core/jobs/playtime_entries.py

```diff
@@ -0,0 +1,134 @@
+import csv
+import os
+from collections import defaultdict
+from datetime import datetime, timedelta
+from tempfile import TemporaryFile
+
+import pytz
+from sqlalchemy.sql.functions import sum
+
+from core.config import Configuration
+from core.model import get_one
+from core.model.edition import Edition
+from core.model.time_tracking import PlaytimeEntry, PlaytimeSummary
+from core.util.datetime_helpers import previous_months, utc_now
+from core.util.email import EmailManager
+from scripts import Script
+
+
+class PlaytimeEntriesSummationScript(Script):
+    def do_run(self):
+        # Reap older processed entries
+        older_than, _ = previous_months(number_of_months=1)
+        older_than_ts = datetime(
+            older_than.year, older_than.month, older_than.day, tzinfo=pytz.UTC
+        )
+        deleted = (
+            self._db.query(PlaytimeEntry)
+            .filter(
+                PlaytimeEntry.processed == True, PlaytimeEntry.timestamp < older_than_ts
+            )
+            .delete()
+        )
+        self.log.info(f"Deleted {deleted} entries. Older than {older_than_ts}")
+
+        # Collect everything from one hour ago, reducing entries still in flux
+        cuttoff = utc_now() - timedelta(hours=1)
+
+        # Fetch the unprocessed entries
+        result = self._db.query(PlaytimeEntry).filter(
+            PlaytimeEntry.processed == False,
+            PlaytimeEntry.timestamp <= cuttoff,
+        )
+        by_identifier = defaultdict(int)
+
+        # Aggregate entries per identifier-timestamp tuple
+        # Since timestamps should be on minute-boundaries the aggregation
+        # can be written to IdentifierPlaytimes directly
```

Comment:
I think this comment needs to be updated to reflect the changes in db table names and facets.